pages/common/banner.md

```diff
@@ -0,0 +1,15 @@
+# banner
+
+> Print large ASCII art version of the given argument.
+
+- Print the text message as a large banner. Note: quotes are optional:
+
+`banner {{"Hello World"}}`
+
+- Print the text message as a banner with a given width:
+
+`banner -w 50 {{"Hello World"}}`
+
+- Read text from the standard input and print it as a large banner:
```

Comment:
Might be easier to use `from stdin` instead of `from the standard input`?

Reply:
Agree :)